Makefile

```diff
@@ -100,6 +101,7 @@ install: all
 uninstall:
 	$(PYTHON) tools/install.py $@ '$(DESTDIR)' '$(PREFIX)'
 
+# Clean the artifacts generated by Makefile in most targets
```

Comment:
I think `clean` is a standard enough makefile target name that this comment shouldn't be necessary. If it is kept, suggest `Remove the artifacts...` reword.

Reply:
This is mostly to distinguish `clean` from `docclean`, `test-addons-clean` and friends.